mkdocs.yml_template

```diff
@@ -215,6 +215,7 @@ nav:
     - "Visualization":
       - "Visualization": gallery/visualization/index.md
       - "COVID Dashboard": gallery/visualization/2_covid_dashboard/index.md
+      - "Measuring Industry Agglomeration": gallery/visualization/industry_agglom/index.md
```

Comment:
The order of cards on the Visualization page differs from the navigation to the left. You can change the navigation or the order of the files by changing the folder name. The cards are ordered alphabetically by their corresponding folders

Reply:
![image](https://github.com/Avaiga/taipy-doc/assets/98709993/36b93c72-a6ae-424d-83c8-2ba8ff5168c5)

Reply:
Fixed by adding "7_" to the folder name